@xen-orchestra/lite/src/stories/web-core/card/vts-card-row-key-value.story.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties, settings }"
+    :params="[
+      slot('key').help('Meant to receive a title'),
+      slot('value').help('Meant to receive text, tags or other value'),
+      slot('addons').help('Meant to receive icons or other actions buttons '),
+      setting('defaultSlotContent').preset('Uuid').widget(text()),
+      setting('contentSlotContent').preset('71df26a271df26a271df26a271df26a271df26a').widget(text()),
+    ]"
+  >
+    <UiCard class="card">
+      <UiCardItem v-bind="properties">
+        <template #key>
+          {{ settings.defaultSlotContent }}
+        </template>
+        <template #value>
+          <span>
+            {{ settings.contentSlotContent }}
+          </span>
+        </template>
+        <template #addons>
+          <VtsIcon accent="warning" :icon="faCircle" :overlay-icon="faStar" />
+          <UiButtonIcon :icon="faCopy" size="medium" accent="info" />
+        </template>
+      </UiCardItem>
+
+      <UiCardItem v-bind="properties">
+        <template #key>
+          <span>Network</span>
+        </template>
+
+        <template #value>
+          <UiObjectLink route="`/vm/test/console`">
+            <template #icon>
+              <UiComplexIcon size="medium">
+                <VtsIcon :icon="faNetworkWired" accent="current" />
+                <VtsIcon accent="success" :icon="faCircle" :overlay-icon="faCheck" />
+              </UiComplexIcon>
+            </template>
+            <span>Network Name</span>
+          </UiObjectLink>
+        </template>
+        <template #addons>
+          <UiButtonIcon :icon="faCopy" size="medium" accent="info" />
+        </template>
+      </UiCardItem>
+
+      <UiCardItem v-bind="properties">
+        <template #key>
+          <span>Status</span>
+        </template>
+        <template #value>
+          <div class="content">
+            <VtsIcon accent="success" :icon="faCircle" :overlay-icon="faCheck" />
+            <span>Connected</span>
+          </div>
```

Comment:
To document the real use case, there is the `UiInfo` component:
```suggestion
          <UiInfo accent="success">Connected</UiInfo>
```